src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Internal/PrecompilationApplication.cs

```diff
@@ -49,5 +51,22 @@ private string GetInformationalVersion()
             var attribute = assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>();
             return attribute.InformationalVersion;
         }
+
+        private static string[] ExpandResponseFiles(string[] args)
```

Comment:
We need to pass in the `ContentFiles` node in rather than scanning for cshtml files to be correct. Setting this up as a first step to pass in args to the tool.